sound/soc/intel/boards/bdw_rt286.c

```diff
@@ -187,13 +187,19 @@ static int card_suspend_pre(struct snd_soc_card *card)
 {
 	struct snd_soc_dai *codec_dai = snd_soc_card_get_codec_dai(card, "rt286-aif1");
 
+	if (!codec_dai)
+		return -EINVAL;
+
```

Comment:
Genuine issue but I'd also expect an error message to show up in dmesg. The fix as I'm reading right now just propagates the error without helping figure out its cause.

Not blocking because of this as even this little can be good.

Reply:
added error message since it is not that urgent.